monitor/census.go

```diff
@@ -725,6 +757,24 @@ func TranscodeTry(nonce, seqNo uint64) {
 	}
 }
 
+func SetTranscodersNumber(transcoders int64) {
```

Comment:
Would it be a little cleaner at the call site to take an `int` type as a parameter, and cast to int64 when calling `Record`? (Same goes for the other functions that were added here)

Reply:
done